src/main/compassus/core.cljc

```diff
@@ -17,7 +17,7 @@
   (-> app :config :root-class))
 
 (defn- make-root-class
-  [{:keys [routes wrapper history]}]
+  [{:keys [routes wrapper wrapper-class history]}]
```

Comment:
wrapper is the component factory. as I understand it, a factory is really only a function. `(om/iquery? (om/react-type wrapper))` returns false.